### PR TITLE
migrate_vm: Enable required ports for migration

### PR DESCRIPTION
**Pull request by dzhengfy**
The firewalld service is started on source host, so migration from
target host to source host will fail due to some required ports
disabled. This is to enable those ports before this kind of migration
is executes.

Signed-off-by: Dan Zheng <dzheng@redhat.com>

### CODE REVIEW

**Comment by dzhengfy** [comment]
Depends on avocado-vt 1132

File: libvirt/tests/src/migration/migrate_vm.py [hunk 1]
```diff
@@ -1104,6 +1104,9 @@ def run(test, params, env):
     # For --migrate-disks test
     migrate_disks = "yes" == test_dict.get("migrate_disks", "no")
 
+    # For bi-directional and tls reverse test
+    src_uri = test_dict.get("migration_source_uri", "qemu:///system")
```
Comment (chunfuwen) [inline libvirt/tests/src/migration/migrate_vm.py]: I don't see migration_source_uri was defined in anywhere

File: libvirt/tests/src/migration/migrate_vm.py [hunk 2]
```diff
@@ -2466,23 +2475,31 @@ def run(test, params, env):
             check_vm_disk_after_migration(vm, test_dict)
 
         if migr_vm_back:
-            src_uri = test_dict.get("migration_source_uri")
+            migrate_setup = libvirt.MigrationTest()
```
Comment (chunfuwen) [inline libvirt/tests/src/migration/migrate_vm.py]: migrate_setup = libvirt.MigrationTest()
migrate_setup.migrate_pre_setup(src_uri, params, ports=uri_port)

those 2 statements were used at least twice,please use function to wrap up them

File: libvirt/tests/src/migration/migrate_vm.py [hunk 3]
```diff
@@ -2466,23 +2475,31 @@ def run(test, params, env):
...
+            # Pre migration setup for local machine
+            migrate_setup.migrate_pre_setup(src_uri, params)
             cmd = "virsh migrate %s %s %s" % (vm_name,
                                               virsh_options, src_uri)
             logging.debug("Start migrating: %s", cmd)
             status, output = run_remote_cmd(cmd, server_ip, server_user,
                                             server_pwd)
+            logging.info(output)
+            # Clean up of pre migration setup for local machine
+            migrate_setup.migrate_pre_setup(src_uri, params,
```
Comment (chunfuwen) [inline libvirt/tests/src/migration/migrate_vm.py]: Contrast with your code on line 2500~2501,it need some conditional cleanup, so a little wonder why here it doesn't need any condition?

File: libvirt/tests/src/migration/migrate_vm.py (hunk 1 above)
Comment (dzhengfy) [reply libvirt/tests/src/migration/migrate_vm.py]: @chunfuwen , migration_source_uri  is in cfg file.

File: libvirt/tests/src/migration/migrate_vm.py (hunk 2 above)
Comment (dzhengfy) [reply libvirt/tests/src/migration/migrate_vm.py]: @chunfuwen updated

File: libvirt/tests/src/migration/migrate_vm.py (hunk 3 above)
Comment (dzhengfy) [reply libvirt/tests/src/migration/migrate_vm.py]: @chunfuwen updated

**Pull request merged**